prov/shm/src/smr_progress.c

```diff
@@ -388,6 +388,8 @@ static int smr_progress_cmd_rma(struct smr_ep *ep, struct smr_cmd *cmd)
 			"unidentified operation type\n");
 		err = -FI_EINVAL;
 	}
+
+comp:
```

Comment:
I'm not sure this is the behavior we want.  IIUC, this is the target of an RMA write which failed it's memory region check.  The receiving side should discard the operation, with only the transmit side (possibly) seeing a failure.

Reply:
Got it. Then I misunderstood. I noticed this when a test was hanging instead of reporting a MR verification failure. I will throw it out then if that's not the expected behavior.